index.html

```diff
@@ -1,139 +1,90 @@
- 
-    <!--About section-->
-    <div class="about-section" id="about">
-        <div id="heading">
-            <h1>Who are we?</h1>
-        </div>
-        <div class="text-container">
-            
-            <div class="paragraph">
-                <p>Greetings from the minds behind the movement! We are Johnny Jones and Will Carlson, the co-founders of Driver’s Club. Born from a shared vision of a more connected and compassionate Europe, we've been steering the wheel towards positive change through our unique rallies.</p>
-            </div>
-            <div class="paragraph">
-                <p>Our journey began with a mutual belief in the power of collective action. As seasoned travelers and advocates, we recognized the potential of rallying communities across Europe to amplify our impact. Driver’s Club emerged as our vessel to turn this vision into a reality.</p>
-            </div>
-        </div>
-        <div class="image-container-pc"> 
-            <div class="person-image">
-                <img src="images/getleman.jpg">
-            </div>
-            <div class="person-image">
-                <img src="images/alfonso-castro-cGqg8Bx2Kbw-unsplash.jpg">
-            </div>
-        </div>
-        <!--Mobile-->
-        <div class="mobile-about">
-            <div class="person-image-mobile">
-                <img src="images/getleman.jpg">
-            </div>
-            <div class="paragraph-mobile">
-                <p>Greetings from the minds behind the movement! We are Johnny Jones and Will Carlson, the co-founders of Driver’s Club. Born from a shared vision of a more connected and compassionate Europe, we've been steering the wheel towards positive change through our unique rallies.</p>
-            </div>
-
-            <div class="person-image-mobile">
-                <img src="images/alfonso-castro-cGqg8Bx2Kbw-unsplash.jpg">
-            </div>
-            <div class="paragraph-mobile">
-                <p>Our journey began with a mutual belief in the power of collective action. As seasoned travelers and advocates, we recognized the potential of rallying communities across Europe to amplify our impact. Driver’s Club emerged as our vessel to turn this vision into a reality.</p>
-            </div>
-        </div>
-
-    </div>
-
-    <!--Cars section-->
-    <div class="cars-section" id="carss">
-        <h1>Cars</h1>
-        <h6>Popular car choices every year.</h6>
-        <div class="image-container-cars">
-            <div class="image-1">
-                <img src="images/flavien-mSC5P8M14BM-unsplash.jpg">
-            </div>
-            <div class="image-2">
-                <img src="images/redcharlie-t8kgpZYWRUE-unsplash.jpg">
-            </div>
-            <div class="image-3">
-                <img src="images/robin-van-geenen-je0BwHBTsm8-unsplash.jpg">
-            </div>
-            <div class="image-4">
-                <img src="images/serjan-midili-yedYysaveC8-unsplash.jpg">
-            </div>
```

Comment:
Insanity. Also, its pretty important to compress images. I like to change their names as well, so I know what is where. You img folder is 30mb, that's mad.  

![image](https://github.com/iCreate1123/Drivers-Club/assets/61486268/5f822b23-4161-498e-aef5-34cbca508886)

Slap a Fast 3G throttle and see how the site loads. Nobody would wait 10 mins to load 5 photos.
Explore the network section in the dev tools, it would really tell you what is slowing down the site the most. It's not too hard to get how it works, and its interesting 😺